pydo/__init__.py

```diff
@@ -0,0 +1,4 @@
+__version__ = 0.1
```

Comment:
Should use strings for versions instead of floats, otherwise one day you'll have something like `v0.30000001`